trackr-app/app/src/main/java/com/trackr/trackr_app/entities/User.kt

```diff
@@ -0,0 +1,9 @@
+package com.trackr.trackr_app.entities
+
+import java.util.UUID
+
+class User (
+        var id: UUID,
```

Comment:
consider using val for the id if it should not be mutated/changed.

Reply:
Ok.